cypress/integration/pages/storyPage/tests.js

```diff
@@ -0,0 +1,25 @@
+import config from '../../../support/config/services';
+
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing features that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
+  describe(`testsThatFollowSmokeTestConfig to run for ${service} ${pageType}`, () => {
+    it('should render a H1, which displays the headline', () => {
```

Comment:
Duplication of https://github.com/bbc/simorgh/blob/f53a329e6f227c379d868886b76745d06a116686/cypress/integration/pages/photoGalleryPage/tests.js#L12

Should be combined in https://github.com/bbc/simorgh/blob/latest/cypress/integration/pages/testsForAllPages.js